MerlinAU.sh

```diff
@@ -4624,7 +4624,7 @@ Please manually update to version $minimum_supported_version or higher to use th
     ## Modified by Martinski W. [2024-Jun-05] ##
     ##----------------------------------------##
     # Fetch the latest SHA256 checksums from ASUSWRT-Merlin website #
-    checksums="$(curl -Ls --retry 4 --retry-delay 5 https://www.asuswrt-merlin.net/download | sed -n '/<pre>/,/</pre>/p' | sed -e 's/<[^>]*>//g')"
+    checksums="$(curl -Ls --retry 4 --retry-delay 5 https://www.asuswrt-merlin.net/download | sed -n '/<p><strong>SHA256 signatures:<\/strong><\/p>/,/<\/pre>/p' | sed -n '/<pre[^>]*>/,/<\/pre>/p' | sed -e 's/<[^>]*>//g' | sed '1d')"
```

Comment:
This line in particular is is the adjusted one for this issue.

Reply:
> This line in particular is is the adjusted one for this issue.
>
I'm thinking of changing the `sed` expressions to make them less dependent on possible future changes in the formatting of the website HTML tags. What do you think? Give me about 20 minutes?

Reply:
I'm all open to it!
My goal was to make it very strict/specific to exclude the betas and non supported.

But I'm also totally open to making this more robust!

Take your time. I'll wait for your final word before prepping a release. Just keep the PRs coming!